ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<workspaceFilter version="1.0">
+    <filter root="/apps/hclecomm"/>
+    <filter root="/content/hclecomm" mode="merge"/>
```

Comment:
@rakeshkumar-verma : <filter root="/content/hclecomm" mode="merge"/> /content/hclecomm this should not part of ui.apps filter xml. should go in content filter.xml

Reply:
removed content filter from ui.apps